frontend/src/app/testResults/uploads/UploadForm.tsx

```
@@ -380,6 +384,22 @@ const UploadForm: React.FC<UploadFormProps> = ({
               uploadType: uploadType,
             },
           });
+        } else {
+          setErrorMessage(
+            <>
+              There was an unexpected processing error. Your file has not been
+              accepted. <br /> Contact support if you continue having issues.
+            </>
+          );
+          setFileValid(false);
+          appInsights?.trackEvent({
+            name: "Unexpected error",
+            properties: {
+              org: orgName,
+              user: user?.email,
+              uploadType: uploadType,
+            },
+          });
```

This avoids a silent failure in the slim chance that there's a 200 response from the backend that doesn't match the first two if statements.